pkg/v1/providers/infrastructure-vsphere/v1.1.0/yttcc/overlay.yaml

```diff
@@ -73,6 +72,29 @@ spec:
         failureDomain: #@ data.values.VSPHERE_AZ_2
         #@ end
       #@ end
+      #@ else:
+      #@overlay/match by=overlay.index(0)
```

Comment:
Is there a better match criteria we can use instead of referencing an index?

Reply:
This one we actually want to make overwrite the 0th element.